cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-settings-page.php

```diff
@@ -161,6 +161,11 @@ private function register_section( $tab_slug ) {
 		register_setting( $setting_slug, $setting_slug, $args );
 		add_filter( 'pre_update_site_option_' . $setting_slug, array( $this, 'set_notices' ), 10, 3 );
 		add_filter( 'pre_update_option_' . $setting_slug, array( $this, 'set_notices' ), 10, 3 );
+
+		if ( ! empty( $tab['javascript_i18n'] ) ) {
+			wp_localize_script( 'jquery', 'cloudinaryJsData', $tab['javascript_i18n'] );
```

Comment:
@dugajean 

Shouldn't we use the dependency on the plugin script instead `cloudinary-settings`?

Reply:
Do we still need this object?